nohup: ignoring input
Traceback (most recent call last):
  File "/data/chy/online/ImageNet/main.py", line 48, in <module>
    model, image_processor, tokenizer = create_model_and_transforms(
  File "/data/chy/online/open_flamingo_4/open_flamingo/src/factory.py", line 51, in create_model_and_transforms
    vision_encoder, _, image_processor = open_clip.create_model_and_transforms(
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/open_clip/factory.py", line 384, in create_model_and_transforms
    model = create_model(
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/open_clip/factory.py", line 203, in create_model
    model = load_openai_model(
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/open_clip/openai.py", line 79, in load_openai_model
    model = model.to(device)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1145, in to
    return self._apply(convert)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/nn/modules/module.py", line 797, in _apply
    module._apply(fn)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/nn/modules/module.py", line 797, in _apply
    module._apply(fn)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/nn/modules/module.py", line 820, in _apply
    param_applied = fn(param)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1143, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/torch/cuda/__init__.py", line 247, in _lazy_init
    torch._C._cuda_init()
RuntimeError: No CUDA GPUs are available
